great_expectations/core/validation_config.py

```diff
@@ -122,6 +134,13 @@ class Config:
     data: BatchConfig = Field(..., allow_mutation=False)
     suite: ExpectationSuite = Field(..., allow_mutation=False)
     id: Union[str, None] = None
+    _validation_results_store: ValidationsStore = PrivateAttr()
+
+    def __init__(self, **data: Any):
+        super().__init__(**data)
+
+        # TODO: Migrate this to model_post_init when we get to pydantic 2
+        self._validation_results_store = project_manager.get_validations_store()
```

Comment:
Can we move this to a read only property (possible cached)?

Reply:
I think we have a slightly inconsistent pattern here - good either way but maybe let's discuss the idiomatic project manager pattern